samples/gcp-storage/src/storageHandler.js

```diff
@@ -0,0 +1,21 @@
+const { App, HTTPBindingMiddleware, StorageMiddleware } = require("core");
+const { GcpModule } = require("gcp");
+const { streamToString } = require("./streamToString");
+
+const app = new App(new GcpModule());
+
+module.exports.handler = app.use(
+  [HTTPBindingMiddleware(), StorageMiddleware()],
+  async (context) => {
+    const { name } = context.req.query;
+    const opts = {
+      container: "storage_sample123",
```

Comment:
The container name is set here and in the yalm. Could you add it to a config file?

Reply:
Done in 64ab69892f5c3ab2cf2a4b060f3d9dc162266d82